Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/GatewayService.java

```diff
@@ -165,8 +174,9 @@ public boolean queryDocument(PatientSearchBean patientQuerySearch) {
      * @param patientQuerySearch
      *
      * @return true if the document retrieval is successful else false
+     * @throws java.io.IOException
      */
-    public boolean retrieveDocument(PatientSearchBean patientQuerySearch) {
+    public boolean retrieveDocument(PatientSearchBean patientQuerySearch) throws IOException {
```

Comment:
why are you throwing exceptions from the managed bean. better to handle them within the method instead of throwing them to the screen.